setup.cfg

```diff
@@ -1,7 +1,7 @@
 [flake8]
 max-line-length=120
 exclude=venv,.venv,bakerydemo/settings/local.py
-ignore=E501,F405,W503
+ignore=E501,F405,W503,E203
```

Comment:
I think this can be reverted now (the banner middleware was the bit of code breaking this rule, and I'm not sure why it had to...)